### PR TITLE
Functional tests - Add new test 'Add to cart by quick view modal'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Add new test 'Add to cart by quick view modal'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Nightly run => https://github.com/nesrineabdmouleh/testing_pr/actions/runs/4204742434
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/FO/home/index.ts [hunk 1]
```diff
@@ -470,15 +476,25 @@ class Home extends FOBasePage {
...
       this.waitForVisibleSelector(page, this.blockCartModalDiv),
       page.click(this.addToCartButton),
     ]);
+
+    return this.getTextContent(page, this.blockCartLabel);
+  }
+
+  /**
+   * Is add to cart button disabled
+   * @param page {Page} Browser tab
```
Comment (Progi1984) [inline tests/UI/pages/FO/home/index.ts:494]:
```suggestion
   * @param page {Page} Browser tab
   * @return {Promise<boolean>}
```

File: tests/UI/campaigns/functional/FO/09_productPage/01_quickView/01_addToCart.ts [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+// Import data
+import Products from '@data/demo/products';
+import CartProductDetails from '@data/types/cart';
+import {ProductAttribute} from '@data/types/product';
+
+import {expect} from 'chai';
+import {BrowserContext, Page} from 'playwright';
+import cartPage from '@pages/FO/cart';
+import searchResultsPage from '@pages/FO/searchResults';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/09_productPage/01_quickView/01_addToCart.ts]:
```suggestion
```

File: tests/UI/campaigns/functional/FO/09_productPage/01_quickView/01_addToCart.ts [hunk 3]
```diff
@@ -0,0 +1,154 @@
+// Import utils
+import helper from '@utils/helpers';
+import testContext from '@utils/testContext';
+
+// Import pages
+import homePage from '@pages/FO/home';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/09_productPage/01_quickView/01_addToCart.ts:6]:
```suggestion
import cartPage from '@pages/FO/cart';
import homePage from '@pages/FO/home';
import searchResultsPage from '@pages/FO/searchResults';
```

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks :) And it's :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**